go/libkb/home.go

```diff
@@ -399,13 +401,13 @@ func (w Win32) MobileSharedHome(emptyOk bool) string {
 	return w.Home(emptyOk)
 }
 
-func NewHomeFinder(appName string, getHomeFromCmd ConfigGetter, getHomeFromConfig ConfigGetter, getMobileSharedHome ConfigGetter, osname string,
+func NewHomeFinder(appName string, getHomeFromCmd ConfigGetter, getHomeFromConfig ConfigGetter, getMobileSharedHome ConfigGetter,
 	getRunMode RunModeGetter, getLog LogGetter, getenv EnvGetter) HomeFinder {
 	base := Base{appName, getHomeFromCmd, getHomeFromConfig, getMobileSharedHome, getRunMode, getLog, getenv}
-	switch osname {
-	case "windows":
+	switch RuntimeGroup() {
+	case keybase1.RuntimeGroup_WINDOWSLIKE:
 		return Win32{base}
-	case "darwin":
+	case keybase1.RuntimeGroup_DARWINLIKE:
```

Comment:
@mmaxim this was the breakage for the mobile build, we'd end up trying to configure ios with linux config paths